sparse/etc/ofono/binder.d/dual-sim.conf

```diff
@@ -0,0 +1,6 @@
+[Settings]
```

Comment:
Single SIM devices will break because of the change in path, one example: https://github.com/mer-hybris/droid-config-sony-ganges-pie/commit/d2e030e710f61e45af6b3c6128d73ace7bbd0007#diff-42e8d3dc485d19351af071057206b835669f1717b8c7205e0ea064ce964f11ccR1

Reply:
Hmm... Instead of adding configuration for the second slot on a dual-SIM device, we install dual-SIM configuration on a single slot device and then delete the dual-SIM part, did I get that right?

Reply:
What it means is that I have to search those delete files for `/etc/ofono/ril_subscription.d/dual-sim.conf` and add `/etc/ofono/binder.d/dual-sim.conf` there, right? It wouldn't be a problem if a file to be deleted doesn't exist, I hope.

Reply:
Yes, you'll have to scour config repos of all devices looking at every delete_file*.list file.
If file isn't there anymore, it won't be a problem at the point of deletion, but wouldn't it break dual SIM functionality of devices, which haven't switched to ofono-binder-plugin but gotten their submodule updated (even if by accident/someone else)?

Reply:
Android 7 and older devices (such as Xperia X, Gemini PDA etc) don't have to be regarded; they will ignore all files under `/etc/ofono/binder.d/` that will appear there after submodule update. They have own /etc/ofono/ril_subscription* in their own [sparses, such as Xperia X](https://github.com/mer-hybris/droid-config-f5121/tree/master/sparse/etc/ofono) and will continue using it.

All devices that are Android 8+ (XA2, and so on) will need to be revised, because some of them had ril conf files under own sparse, and others relied on sparse-10 and sparse-11, that you have rightly nuked in this PR.